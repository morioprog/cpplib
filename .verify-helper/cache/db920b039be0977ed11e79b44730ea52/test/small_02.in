3 20
158175838295 -712859138678 851132932613
1 0 1 751437114362
2 2 3 -211595077827
1 0 2 995648468782
1 0 2 -261334714485
2 1 3 -255622399121
0 2 3 682560499495
3 0 2
3 1 3
1 0 2 1406229528
2 2 3 -591700260452
0 0 2 -129109003562
0 0 2 110590608714
0 1 2 466452067774
3 1 2
2 0 3 -87213933184
3 1 2
2 0 1 993721463328
1 2 3 -450166040784
0 0 2 -688387147441
3 1 2
