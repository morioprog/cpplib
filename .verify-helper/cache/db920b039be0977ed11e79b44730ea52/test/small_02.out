1735674538443
1123941525326
-129109003562
-216322936746
-688387147441
